core/math/basis.cpp

```diff
@@ -691,6 +691,10 @@ bool Basis::is_equal_approx(const Basis &p_basis) const {
 	return rows[0].is_equal_approx(p_basis.rows[0]) && rows[1].is_equal_approx(p_basis.rows[1]) && rows[2].is_equal_approx(p_basis.rows[2]);
 }
 
+bool Basis::is_finite() const {
+	return rows[0].is_finite() && rows[1].is_finite() && rows[2].is_finite();
+}
```

Comment:
Considering you can validate a Basis by checking its determinant, I'm not sure this is helpful. The same likely applies to several others. But I guess it's fine.